samples/sensor/am2301b/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: AM2301B Sensor sample
+tests:
+  sample.sensor.am2301b:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840
+    extra_args: "DTC_OVERLAY_FILE=arduino_i2c.overlay"
+    depends_on: i2c am2301b
```

Comment:
this sample is not exclusive to nrf, please filter using `dt_compat_enabled` (see some other samples).

another thing: you can use `app.overlay`, which is automatically picked.

Reply:
fixed it.